conbench/tests/api/test_runs.py

```diff
@@ -21,7 +23,7 @@ def _expected_entity(run, baseline_id=None, include_baseline=True):
         run.hardware_id,
         run.hardware.name,
         run.hardware.type,
-        run.timestamp.isoformat(),
+        tznaive_dt_to_aware_iso8601_for_api(run.timestamp),
```

Comment:
Just FYI. Without this change, the corresponding test failure would be something like
```
E         -  'timestamp': '2023-01-31T18:10:31.686148',
E         ?                                   ^^^^^^^
E         +  'timestamp': '2023-01-31T18:10:31Z',
```

The first line is the return value if `run.timestamp.isoformat()` (showing fractions of seconds and _no_ timezone information).

The second line is the return value of the new `tznaive_dt_to_aware_iso8601_for_api()`.